.github/build-php-fpm-image.sh

```diff
@@ -0,0 +1,11 @@
+docker image build \
+    --build-arg project_root=project-base \
+    --build-arg www_data_uid=$(id -u) \
+    --build-arg www_data_gid=$(id -g) \
+    --tag ${DOCKER_USERNAME}/php-fpm:${DOCKER_PHP_FPM_IMAGE_TAG} \
+    --target development \
+    --no-cache \
+    --compress \
+    -f project-base/docker/php-fpm/Dockerfile \
+    . &&
+docker image push ${DOCKER_USERNAME}/php-fpm:${DOCKER_PHP_FPM_IMAGE_TAG}
```

Comment:
nitpick:
The tag could be extracted into variable as it is used on line 5 in a same way